Doc/whatsnew/3.7.rst

```diff
@@ -282,7 +282,14 @@ Other Language Changes
 New Modules
 ===========
 
-* None yet.
+importlib.resources
```

Comment:
Can we use ```:mod:`importlib.resources` ``` in titles so it links to the documentation?

Reply:
None of the other entries in this section (i.e. the "Improved Modules") uses links, so I just went with consistency here.